template/README.md

```diff
@@ -346,6 +346,8 @@ git checkout -
 
 You may copy and paste them, or put them into a custom shell script. You may also customize them for another hosting provider.
 
+Note that if you add a router your app won't work with GitHub Pages becasue it doesn't natively support single page apps with frontend routing (unless it's the ugly hashtag kind). There is a simple workaround for this by adding a `404.html` file with a redirect script, see [Single Page Apps for GitHub Pages](https://github.com/rafrex/spa-github-pages).
```

Comment:
Let’s make it clear that it’s not an inherent problem with routers. It’s only relevant for the routers using HTML5 `pushState` history API under the hood. (I know you know it, but our readers probably won’t.)